### PR TITLE
Remove Array print statements

### PR DESCRIPTION
**Pull request by 9prady9**

### CODE REVIEW

**Comment by 9prady9** [comment]
build arrayfire windows ci

File: test/index.cpp [hunk 1]
```diff
@@ -536,9 +536,6 @@ TEST(Index, Docs_Util_C_API)
 
     err = af_set_seq_indexer(indexers, &zeroIndices, 0, false);
 
-    af_print_array(a);
-    af_print_array(out);
-
```
Comment (pavanky) [inline test/index.cpp]: Is this a release statement?

Comment (9prady9) [inline test/index.cpp]: @pavanky Addressed it. Thank you.

**Comment by shehzan10** [comment]
build arrayfire windows cpu ci

**Pull request merged**